string[] name
bool[] save_current_settings
string[] set_name_part
string[] set_family_part
int8[] control_strategy
hebiros/PidGainsMsg position_gains
hebiros/PidGainsMsg velocity_gains
hebiros/PidGainsMsg effort_gains

